src/main/resources/internal-admin-api.yaml

```diff
@@ -1464,6 +1464,11 @@ definitions:
       selfRegistrationAllowed:
         type: boolean
         description: Flag indicating whether Self-service registration is enabled for this service.
+      ssoProviders:
```

Comment:
Is it worth making this param a separate type as it seems it's used twice in the exact same form?

Reply:
it shows up twice because the joker who designed this in the first place made user and the object you post to update a user.. two different, almost identical objects..

really good